src/js/components/AutolinkComponent.jsx

```diff
@@ -0,0 +1,56 @@
+var React = require("react/addons");
+
+/**
```

Comment:
Yeah, but we modified the autolink that much to our needs, that I wouldn't replace it. It's better then the original one. :)